packages/dcos-integration-test/extra/integration_test.py

```diff
@@ -1105,6 +1105,79 @@ def _ensure_routable():
     _ensure_routable()
 
 
+def test_if_minuteman_routes_to_named_vip(cluster, timeout=125):
+    """Test if we are able to connect to a task with a vip using minuteman.
+    """
+    # Launch the app and proxy
+    test_uuid = uuid.uuid4().hex
+
+    app_definition = {
+        'id': "/integration-test-app-with-minuteman-vip-%s" % test_uuid,
```

Comment:
we have a standard marathon app we launch, can we just use that + add VIP config to it?